satellite/gracefulexit/endpoint.go

```diff
@@ -626,6 +637,41 @@ func (endpoint *Endpoint) handleFailed(ctx context.Context, pending *pendingMap,
 	return nil
 }
 
+func (endpoint *Endpoint) getFinishedMessage(ctx context.Context, signer signing.Signer, nodeID storj.NodeID, finishedAt time.Time, success bool, reason pb.ExitFailed_Reason) (message *pb.SatelliteMessage, err error) {
```

Comment:
In master now there is a comment `// TODO(moby) use getFinishedMessage helper after 3368 is merged`
Could you make sure to remove the comment and call `getFinishedMessage` in this PR?

Reply:
updated